### PR TITLE
Fix size hot key for RAID tests

### PR DESCRIPTION
**Pull request by rwx788**
After code refactoring I've missed adding $cmd{size_hotkey} so test
started to fail as we do not press anything.

### CODE REVIEW

File: tests/installation/partitioning.pm [hunk 1]
```diff
@@ -29,6 +29,7 @@ sub run {
         $cmd{addraid}         = 'alt-i';
         $cmd{filesystem}      = 'alt-a';
         $cmd{exp_part_finish} = 'alt-n';
+        $cmd{size_hotkey}     = 'alt-s';
```
Comment (kravciak) [inline tests/installation/partitioning.pm]: Maybe I am wrong, but "Define changed shortcuts" comment - is this needed here if susedistribution.pm has the same shortcut?

Comment (Soulofdestiny) [reply tests/installation/partitioning.pm]: where did you spot this in susedistribution?

Comment (kravciak) [reply tests/installation/partitioning.pm]: This PR is adding 2 lines of code - I am wondering if first line is not enough

Comment (Soulofdestiny) [reply tests/installation/partitioning.pm]: ah now I got it, @rwx788 will handle accordingly

Comment (rwx788) [reply tests/installation/partitioning.pm]: My bad. @kravciak Thanks a lot!

**Comment by okurz** [comment]
LGTM

**Pull request merged**